### PR TITLE
docs(policies): add guide for metrics collection with opentelemetry

### PR DESCRIPTION
**Pull request by Automaat**
Fixes: #1677 

It would be nice if someone do this guide and check if everything works as expected

### CODE REVIEW

**Comment by slonka** [comment]
Rendered: https://deploy-preview-1718--kuma.netlify.app/docs/dev/guides/otel-metrics/

File: app/_src/guides/otel-metrics.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+---
+title: Collect metrics with OpenTelemetry  
+---
+
+{{site.mesh_product_name}} provides integration with [OpenTelemetry](https://opentelemetry.io/). You can collect and push 
+dataplane proxy and application metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/). Which opens up
+lots of possibilities of processing and exporting metrics to multiple ecosystems like [Datadog](https://www.datadoghq.com/),
+[Grafana cloud](https://grafana.com/products/cloud/), [Honeycomb](https://www.honeycomb.io/) and more.
```
Comment (slonka) [inline app/_src/guides/otel-metrics.md:8]: should we mention specific implementations or just link to the exporters listing https://opentelemetry.io/ecosystem/registry/?component=exporter ?

File: app/_src/guides/otel-metrics.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+Then navigate to browser `http://localhost:3000` and check `Dataplane` dashboard. You should see something similar to:
+
+<center>
+<img src="/assets/images/guides/otel-metrics/grafana-dataplane-view.png" alt="Dataplane Grafana dashboard"/>
+</center>
+
+## Next steps
+
+* Further explore [MeshMetric policy](/docs/{{ page.version }}/policies/meshmetric)
```
Comment (slonka) [inline app/_src/guides/otel-metrics.md:153]: should we link guides how to export this to hosted solutions?

File: app/_src/guides/otel-metrics.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+By now we have installed and configured all needed observability tools: OpenTelemetry collector, Prometheus and Grafana.
+We can now apply [MeshMetric policy](/docs/{{ page.version }}/policies/meshmetric):
+
+```shell
+echo "
+apiVersion: kuma.io/v1alpha1
+kind: MeshMetric
+metadata:
+  name: 'otel-metrics'
+  namespace: kuma-system
```
[the remark this answers is not in the document]
Comment (Automaat) [reply app/_src/guides/otel-metrics.md]: changed

File: app/_src/guides/otel-metrics.md (hunk 2 above)
Comment (Automaat) [reply app/_src/guides/otel-metrics.md:153]: good idea, I've added links to tutorials

File: app/_src/guides/otel-metrics.md (hunk 1 above)
Comment (Automaat) [reply app/_src/guides/otel-metrics.md:8]: we have this link later on when explaining configuration, I think that it should be enough

File: app/_src/guides/otel-metrics.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+<center>
+<img src="/assets/images/guides/otel-metrics/grafana-dataplane-view.png" alt="Dataplane Grafana dashboard"/>
+</center>
+
+## Next steps
+
+* Further explore [MeshMetric policy](/docs/{{ page.version }}/policies/meshmetric)
+* Explore [MeshAccessLog](/docs/{{ page.version }}/policies/meshaccesslog/#opentelemetry) and [MeshTrace](/docs/{{ page.version }}/policies/meshtrace/#opentelemetry) policies that work with OpenTelemetry
+* Explore features of [OpenTelemetry collector](https://opentelemetry.io/docs/collector/) for metrics filtering/processing and exporting
+* Checkout tutorials on how to push metrics to saas solutions [Grafana cloud](https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/setup/collector/opentelemetry-collector/) [Datadog](https://www.datadoghq.com/blog/ingest-opentelemetry-traces-metrics-with-datadog-exporter/)
```
Comment (slonka) [inline app/_src/guides/otel-metrics.md:156]:
```suggestion
* Checkout tutorials on how to push metrics to SaaS solutions [Grafana cloud](https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/setup/collector/opentelemetry-collector/), [Datadog](https://www.datadoghq.com/blog/ingest-opentelemetry-traces-metrics-with-datadog-exporter/), [Honeycomb](https://docs.honeycomb.io/send-data/opentelemetry/collector/)
```

**Review by slonka — APPROVED** [review]

**Pull request merged**